src/renderer.js

```diff
@@ -108,8 +110,21 @@ export default function Emoji({
   onlyEmojiClassName,
   options = {},
   className,
+  children,
   ...rest
 }) {
+  let non_string_elements = [];
+
+  if (!!children && Array.isArray(children)) {
+    const [strippedChildren, elements] = stripNonStringElements(children);
+    text = strippedChildren;
+    non_string_elements = elements;
+  } else if (!!children && typeof children === "string") {
+    text = children;
+  } else if (!text) {
+    throw new Error("Emoji: either children or text prop must be provided");
```

Comment:
How about using the library name as a prefix ? So it's easier to spot:

```suggestion
    throw new Error("react-emoji-render: either children or text prop must be provided");
```